build/config.go

```diff
@@ -66,25 +65,11 @@ func init() {
 	wd, _ := os.Getwd()
 	WorkloadPath = filepath.Join(wd, "templates", "oam", "workloads")
 	MeshModelPath = filepath.Join(wd, "templates", "meshmodel", "components")
-	vs, err := config.GetLatestReleaseNames(30)
-	if len(vs) == 0 {
-		fmt.Println("dynamic component generation failure: ", err.Error())
+	AllVersions, _ = utils.GetLatestReleaseTagsSorted("istio", "istio")
```

Comment:
Please use linkerd/linkerd2 instead of istio

Reply:
It should be updated now.